service/build.gradle

```diff
@@ -53,9 +53,9 @@ dependencies {
 	// Allows us to mock final classes
 	testImplementation 'org.mockito:mockito-inline:5.2.0'
 
-	testImplementation 'au.com.dius.pact.consumer:junit5:4.1.7'
-	testImplementation("au.com.dius.pact.provider:junit5:4.1.7")
-	testImplementation("au.com.dius.pact.provider:junit5spring:4.1.7")
+	testImplementation 'au.com.dius.pact.consumer:junit5:4.6.1'
```

Comment:
The point release dependabot PR was failing. I decided to go to the newest version while fixing the compatibility issues.